visualization/tools/pcd_viewer.cpp

```diff
@@ -617,22 +617,25 @@ main (int argc, char** argv)
     {
       int rgb_idx = 0;
       int label_idx = 0;
+      int invalid_fields_count = 0;
       for (size_t f = 0; f < cloud->fields.size (); ++f)
       {
+        if (!isValidFieldName (cloud->fields[f].name)) {
```

Comment:
Please put the curly bracket on a separate line

Reply:
sorry about that. I'm not used to this convention. Do you use some clang-format/linter?

btw do you know there are many trailing whitespace issues?